Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -342,7 +342,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   CreateVobs: 1
-  CreateWaypoints: 0
+  CreateWaypoints: 1
```

Comment:
Please revert. I commented insige WayNetCreator.cs how to change behaviour. This debug setting is for visualizing WayPoints only. Not for "having them" ;-)

Reply:
i use them to be able to teleport the player to more points, since the teleport scrolls teleport the player to waypoints, not freepoints, i added them to the list (as discussed)